intel-sgx/scripts/gdb.py

```diff
@@ -30,8 +38,51 @@ def read_long(inferior, address, length):
     ret += b
   return ret
 
-def find_vma_base(addr):
-  s = gdb.execute("info proc mappings", False, True)
+# def get_current_language():
+#   """Get the current GDB language, as seen by `show language` and `info frame`.
+
+#   (Ironically, `info frame` seems easier to parse.)
+#   """
+#   info_frame = gdb.execute(f"info frame", from_tty = False, to_string = True)
+#   for line in info_frame.splitlines():
+#     # GDB does not seem to support non-English locales.
+#     # Which is hopefully true on your machine as well,
+#     # because we're hardcoding the text :see_no_evil:
+#     match=re.match('^\s*source language ([0-9a-zA-Z-]+).$', line)
+#     if match is not None:
+#       return match.group(1)
+#       break
+#   print("No language was found in `info frame`. That should not be possible.")
```

Comment:
I assume this should be removed?

Reply:
Yess, that is correct. I've later found a better alternative approach, and this is no longer needed.